### ISSUE TITLE
Add "titles" to STPA viewpoint

### ISSUE THREAD

**Issue by rreis** [labs4capella/stpa-capella#5]
It would be nice to have tiles available in STPA viewpoint so we could track connections of the diagrams and other info. It would provide extra support to the analysis.

**Comment by oconstant** [comment]
Indeed, Title Blocks would probably be as relevant as in classical Capella perspectives.
